cfgov/legacy/templates/front/base_update.html

```diff
@@ -86,36 +109,43 @@
     {% endblock %}
 
     <!--[if lt IE 9]>
-    <link rel="stylesheet" href="{% static 'css/header.nonresponsive.css' %}"/>
-    <link rel="stylesheet" href="{% static 'css/footer.nonresponsive.css' %}"/>
+    <link rel="stylesheet" href="{{ STATIC_PREFIX }}css/header.nonresponsive.css"/>
+    <link rel="stylesheet" href="{{ STATIC_PREFIX }}css/footer.nonresponsive.css"/>
     <![endif]-->
 
     <!--[if gt IE 8]><!-->
-    <link rel="stylesheet" href="{% static 'css/header.css' %}"/>
```

Comment:
Okay, I removed `STATIC_PREFIX` in https://github.com/cfpb/cfgov-refresh/pull/2283/commits/f211e66d087745bd2dba7099808cbc0318e0006d ... please check over. I didn't update to the `static` tag on those links that used `STATIC_VERSION` as I'm not sure what the syntax would look like.